py_gtfs_rt_ingestion/lib/convert_gtfs_rt.py

```diff
@@ -30,8 +33,8 @@ class GtfsRtConverter(Converter):
     https_mbta_integration.mybluemix.net_vehicleCount.gz
     """
 
-    def __init__(self, config_type: ConfigType) -> None:
-        Converter.__init__(self, config_type)
+    def __init__(self, config_type: ConfigType, files: List[str]) -> None:
+        Converter.__init__(self, config_type, files)
```

Comment:
I think it would be worth adding the pyarrow filesystem initialization to `__init__` . it's another fairly expensive object creation that we are performing for every gzip that we ingest. 

I did some profiling and got between 35% and 40% improvement in bulk file downloading speed with just one `fs` creation per thread.

Reply:
i addressed this. we initialize a local filesystem first, and at the start of processing files, change it if the first file has an s3 prefix. that filesystem is used for all reading.